streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableSourceTest.java

```diff
@@ -148,8 +153,7 @@ public void testNotSendingOldValue() {
         final StreamsBuilder builder = new StreamsBuilder();
         final String topic1 = "topic1";
 
-        @SuppressWarnings("unchecked")
-        final KTableImpl<String, String, String> table1 = (KTableImpl<String, String, String>) builder.table(topic1, stringConsumed);
+        @SuppressWarnings("unchecked") final KTableImpl<String, String, String> table1 = (KTableImpl<String, String, String>) builder.table(topic1, stringConsumed);
```

Comment:
line too long -- why reformatting?

Reply:
Sorry, but i don't get this erros in style formatting. Probably if some on could update the style checker file.

Reply:
I disabled all IntelliJ auto-reformatting to avoid any unrelated reformatting.